### PR TITLE
Add-writing-prompts

### PR DESCRIPTION
**Pull request by jaresty**
Try this:
- use dictation mode to brainstorm a rough topic idea
- use model add questions to add questions to it and answer the questions one by one
- use model format outline to generate a rough outline for that
- use model format prose to take that outline and turn it into text that reads well

### CODE REVIEW

File: GPT/lists/staticPrompt.talon-list [hunk 1]
```diff
@@ -48,3 +48,8 @@ generate code: The following plaintext describes a process in code in the langua
...
 clean code: Act as a software engineer. Reduce any duplication in the selected code and improve it to be more idiomatic and clear for other users. However, do not change the behavior or functionality. Return just the code and not any explanations.
 improve semantics: The following is an HTML document. Keep the same structure and layout but if it is needed, change any elements to use proper semantic HTML and make sure it is implementing best practices for user accessibility. Output just the HTML and not any extra explanations.
+
+## PROMPTS TO HELP WITH WRITING
+add questions: Help me explore this question from multiple perspectives. For each perspective, ask follow-up questions and indicate what perspective is being taken. Annotate the dialogue to show its a conversation between you and me, as this will be shared on Slack and needs to be clear that its a generated response. Respond with only the perspectives and the questions they generated.
```
Comment (C-Loftus) [inline GPT/lists/staticPrompt.talon-list:53]:
```suggestion
add questions: Help me explore this question from multiple perspectives. For each perspective, ask follow-up questions and indicate what perspective is being taken. 
```
I think the part about having it annotated as a dialog for sharing with others might be too specific.

File: GPT/lists/staticPrompt.talon-list [hunk 2]
```diff
@@ -48,3 +48,8 @@ generate code: The following plaintext describes a process in code in the langua
...
+format outline: Create an outline that encapsulates the text below. Keep the number of sections between three and five to optimize for human working memory. Return just the outline.
+format prose: As an editor, format the following text as prose. You can have headings and paragraphs. Trying to avoid bullets. Reorder and add transitions as necessary to make the document flow. Return just the text.
```
Comment (C-Loftus) [inline GPT/lists/staticPrompt.talon-list:55]: When would this be used?

Comment (C-Loftus) [inline GPT/lists/staticPrompt.talon-list:55]:
```suggestion
format prose: As an editor, format the following text as prose. You can have headings and paragraphs. Avoid using bullet points. Reorder and add transitions as necessary to make the document flow. Return just the text.
```

File: GPT/lists/staticPrompt.talon-list [hunk 3]
```diff
@@ -48,3 +48,8 @@ generate code: The following plaintext describes a process in code in the langua
...
 clean code: Act as a software engineer. Reduce any duplication in the selected code and improve it to be more idiomatic and clear for other users. However, do not change the behavior or functionality. Return just the code and not any explanations.
 improve semantics: The following is an HTML document. Keep the same structure and layout but if it is needed, change any elements to use proper semantic HTML and make sure it is implementing best practices for user accessibility. Output just the HTML and not any extra explanations.
+
+## PROMPTS TO HELP WITH WRITING
+add questions: Help me explore this question from multiple perspectives. For each perspective, ask follow-up questions and indicate what perspective is being taken. Annotate the dialogue to show its a conversation between you and me, as this will be shared on Slack and needs to be clear that its a generated response. Respond with only the perspectives and the questions they generated.
+format outline: Create an outline that encapsulates the text below. Keep the number of sections between three and five to optimize for human working memory. Return just the outline.
```
Comment (C-Loftus) [inline GPT/lists/staticPrompt.talon-list:54]: A bit similar to summarize but I think it is ok.

**Comment by C-Loftus** [comment]
Fine with me. Made a few suggestions. I think there are a few ways in which we can make it more generalizable beyond our specific workflows. I think the pattern of adding questions, summarizing into an outline, and then condensing is a fair pattern.

**Comment by jaresty** [comment]
Let me give the new prompts a whirl before we merge-I want to make sure they're still working well.

**Comment by jaresty** [comment]
Okay I tested it out and it seems to be working still. Works for me!

**Review by C-Loftus — APPROVED** [review]

**Pull request merged**